dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
>@weshaggard , this is scoped to the package .builds files. You're saying that even that is too much?

What is scoping it to that? In the end it likely doesn't matter but it might be worth making this target only apply to packages.builds.